src/kata6.joinNames.js

```diff
@@ -1,5 +1,11 @@
 const joinNames = (namesObj) => {
-
-}
+    const namesArr = [];
+for( let i = 0; i< namesObj.length; i++){
+namesArr.push(namesObj[i]);
+console.log(namesArr);
+};
```

Comment:
The above code is all redundant. `namesObj` is already an array - this code just takes the array and makes a new (identical) array.

Reply:
Please ensure also that you don't commit console.log statements :)